api/src/main/java/io/grpc/ServerInterceptors.java

```diff
@@ -291,23 +291,23 @@ public ServerCall.Listener<WReqT> startCall(
           final Metadata headers) {
         final ServerCall<OReqT, ORespT> unwrappedCall =
             new PartialForwardingServerCall<OReqT, ORespT>() {
-          @Override
-          protected ServerCall<WReqT, WRespT> delegate() {
-            return call;
-          }
+              @Override
```

Comment:
Have to leave this as is - diamond notation does not work with anonymous classes in java 8.